app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    id: str
```

Comment:
Consider changing the type of `id` from `str` to `int` if the database schema uses integers for IDs. This will ensure consistency between the model and the database.